conftest.py

```diff
@@ -21,6 +21,7 @@ def pytest_addoption(parser: Parser) -> None:
     aws_group = parser.getgroup(name="AWS")
     buckets_group = parser.getgroup(name="Buckets")
     runtime_group = parser.getgroup(name="Runtime Details")
+    upgrade_group = parser.getgroup(name="Upgrade options")
```

Comment:
options -> Options

Reply:
modified `Runtime Details` to `Runtime details`